cki-send.py

```diff
@@ -55,12 +43,21 @@ def main(args):
     for (x, y) in instances:
         filename = f"{x}.{y}.result"
         result = pathlib.Path(running_path / filename ).read_text().strip()
-        new_temp = template.copy()
-        new_temp["test_name"] = f"{args.cloud}-{x}"
-        new_temp["test_description"] = f"Kernel test result on instance {x} of {args.cloud}"
-        new_temp["test_arch"] = y
-        new_temp["test_result"] = result
-        msg["results"].append(new_temp)
+        msg["tests"].append({
+            "build_id": args.kcidb_build_id,
+            "id": f'redhat:virt_qe_s1_{args.build_id}_{x}',
+            "origin": "redhat",
+            "environment": {
+                "comment": f'Instance {x} of {args.cloud}'
+            },
+            "path": "ltp"
+            "log_url": f'{args.build_url}/consoleText',
+            "status": result,
```

Comment:
Is `result in ["ERROR", "FAIL", "PASS", "DONE", "SKIP"]`?

Reply:
@imalerba if result is `ABORT` (set here https://github.com/virt-s1/cki/blob/master/cloudci/Jenkinsfile.openstack#L194) should I map that to `FAIL`?

Reply:
Actually I think I'll just use results for the subtests rather than the summary result, which is either `PASS` or `FAIL`: https://github.com/virt-s1/cki/blob/master/templates/ltp_report.j2#L2

Reply:
Yes, we need to convert the result to one of the KCIDB accepted :)

https://gitlab.com/cki-project/cki-tools/-/blob/main/cki/kcidb/objects.py#L995